apps/web/src/routes/(home)/components/CtaBlock.svelte

```diff
@@ -45,6 +46,33 @@
 	function handleClickOutside() {
 		showSelect = false;
 	}
+
+	async function detectDefaultDownload() {
+		const platform = navigator.userAgent.toLowerCase();
+
+		if (platform.includes('mac')) {
+			const userAgentData = await navigator.userAgentData?.getHighEntropyValues(['architecture']);
+			if (userAgentData?.architecture === 'arm' || userAgentData === undefined) {
```

Comment:
If these terms were swapped I could imagine TS being smart enough to allow doing `userAgentData.architecture`.